features/support/env.js

```diff
@@ -14,6 +15,10 @@ class CustomWorld {
   constructor() {
     this.driver = driver;
   }
+
+  testId() {
```

Comment:
There may already be something like this in cucumber, but I didn't take the time to find out :X